src/gpgi/_lib.pyx

```diff
@@ -1,3 +1,5 @@
+# cython: freethreading_compatible = True
```

Comment:
This macro is not documented yet, so I'll wait until https://github.com/cython/cython/pull/6298 is merged before I can call it stable enough

Reply:
It's documented in https://cython.readthedocs.io/en/latest/src/userguide/source_files_and_compilation.html#compiler-directives already.

(With that said, you should the official line is that everything free-threaded is experimental for the foreseeable future and may change as needed. But that particular directive is probably fairly unlikely to change)

Reply:
thanks, I missed that ! And dully noted, all of this is still experimental, I'm just experimenting along :)